src/triggers/default_triggers.py

```diff
@@ -11,30 +11,14 @@
         "remote_deployment": Trigger(
             conditions=[lambda config: config.remote_service_url is not None], worker_disk=consts.DISK_SIZE_100GB
         ),
-        "none_platform": Trigger(
-            conditions=[lambda config: config.platform == consts.Platforms.NONE],
-            user_managed_networking=True,
-            tf_platform=consts.Platforms.NONE,
-        ),
-        "vsphere_platform": Trigger(
-            conditions=[lambda config: config.platform == consts.Platforms.VSPHERE],
-            user_managed_networking=False,
-            tf_platform=consts.Platforms.VSPHERE,
```

Comment:
I guess somewhere else we're matching the terraform platform with the ocp platform?
Or we still need it just like the nutanix trigger below?

Reply:
There is an issue here that I'm trying to solve, the case where we pass `PLATFORM=...` as an env var is already solved on the `env_variables_defaults.py` file. But we are not handling the case where we pass platform as a parametrize parameter.
The problem there is that we triggers are called after the node controller configuration is created but the configuration is created by a given `tf_platform`.
So for conclusion we don't need the triggers set for `tf_platform` and I'm working on fixing the other issue (I missed nutanix for some reason and I'll remove it on the next push)